README.md

```diff
@@ -1,24 +1,249 @@
-# README
+<a name="readme-top"></a>
```

Comment:
### I'm just using the readme file to write comments because not all files are included in this PR you should modify the code that is responsible for what is shown below 

- One of the requirements is to follow the given design by implementing the same colors, font-family, spaces, and layout you can see the screenshot of the requirements below 
<img width="546" alt="Screenshot 2023-02-15 at 15 24 42" src="https://user-images.githubusercontent.com/61790587/219056526-1c6f88ad-759e-45a9-805d-f8aa0f2e4ded.png">

- [ ] it would be better if you make the home page as in the Design that has two buttons (sign up and login )
- Home page design 
<img width="236" alt="Screenshot 2023-02-15 at 15 26 59" src="https://user-images.githubusercontent.com/61790587/219056997-d2289806-57fe-4113-969c-155dd2103b20.png">